src/app/tabs/run-tab.js

```diff
@@ -10,6 +10,7 @@ var executionContext = require('../../execution-context')
 var copyToClipboard = require('../ui/copy-to-clipboard')
 var Recorder = require('../../recorder')
 var EventManager = require('remix-lib').EventManager
+var ethereumjsUtil = require('ethereumjs-util')
```

Comment:
I think the usual name for this is `ethUtil`, but please search for other `require('ethereumjs-util')` statements and follow the naming fomr there.

Reply:
@axic -- There are 2 naming conventions : `ethUtil` and `ethJSUtil`. I'm changing it to `ethJSUtil` as it's being used in majority of the places